beets/util/artresizer.py

```diff
@@ -219,9 +231,12 @@ def _can_compare(self):
     @staticmethod
     def _check_method():
         """Return a tuple indicating an available method and its version."""
```

Comment:
This docstring could use a little expansion to describe what the method now returns.